dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/ndk/NdkCrashLog.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.ndk
+
+import com.google.gson.JsonObject
+import com.google.gson.JsonParseException
+import com.google.gson.JsonParser
+
+internal data class NdkCrashLog(
+    val signal: Int,
+    val timestamp: Long,
+    val signalName: String,
+    val message: String,
+    val stacktrace: String
+
+)
+
+@Throws(JsonParseException::class)
+internal fun fromJson(jsonString: String): NdkCrashLog {
+    val jsonObject = JsonParser.parseString(jsonString).asJsonObject
+    return NdkCrashLog(
+        jsonObject.get(SIGNAL_KEY_NAME).asInt,
+        jsonObject.get(TIMESTAMP_KEY_NAME).asLong,
+        jsonObject.get(SIGNAL_NAME_KEY_NAME).asString,
+        jsonObject.get(MESSAGE_KEY_NAME).asString,
+        jsonObject.get(STACKTRACE_KEY_NAME).asString
+
+    )
+}
+
+internal fun NdkCrashLog.toJson(): String {
```

Comment:
You have the Class defined just above, it would make sense to make this a method from the class rather than an extension one.